### ISSUE TITLE
video missing description of visual text

### ISSUE THREAD

**Issue by shawna-slh** [w3c/w3c-website#494]
**Describe the issue**
The video includes visual information that is not available to people who cannot see.
beginning:
> What part of the World Wide Web Consortium's work are you nost proud of?
> WORLD WIDE WEB FOUNDATION

throughout:
> Sir Tim Berners-Lee
> Web Inventor
> W3C

end:
> #WEB30
> WORLD WIDE WEB FOUNDATION

**URL**
https://www.w3.org/mission/

**Recommended solution**
Ideally that info would be provided as a separate voice-over at the beginning and ending of the video. (for WCAG AA conformance)

In this situation, I think it's acceptable to put that as static text with the video, per sufficient technique [G203: Using a static text alternative to describe a talking head video](https://www.w3.org/WAI/WCAG21/Techniques/general/G203)

Perhaps:
> Sir Tim Berners-Lee, Web Inventor, W3C, answers the question "What part of the World Wide Web Consortium's work are you most proud of?" in this video from the World Wide Web Foundation. #WEB30

**Additional info**
[Description of Visual Information, in Making Audio and Video Media Accessible](https://www.w3.org/WAI/media/av/description/)

**Comment by japuiglaucia** [comment]
video missing description of visual text #494

**Comment by koalie** [comment]
Done!
